lib/features/composer/presentation/composer_controller.dart

```diff
@@ -716,14 +719,19 @@ class ComposerController extends BaseController with DragDropFileMixin implement
     }
   }
 
+  Identity? _selectedIdentityFromId(IdentityId? identityId) {
+    if (identityId == null) return null;
+
+    return listFromIdentities.firstWhereOrNull(
+      (identity) => identity.id == identityId);
+  }
+
   void _initIdentities(ComposerArguments composerArguments) {
     listFromIdentities.value = composerArguments.identities ?? [];
     if (listFromIdentities.isEmpty) {
       _getAllIdentities();
-    } else if (composerArguments.selectedIdentity != null
-      && listFromIdentities.contains(composerArguments.selectedIdentity!)
-    ) {
-      _selectIdentity(composerArguments.selectedIdentity!);
+    } else if (_selectedIdentityFromId(composerArguments.selectedIdentityId) != null) {
+      _selectIdentity(_selectedIdentityFromId(composerArguments.selectedIdentityId));
```

Comment:
please use assignment to avoid duplicate `_selectedIdentityFromId(composerArguments.value?.selectedIdentityId)`

Reply:
Done